snapcraft/models/project.py

```diff
@@ -372,7 +372,7 @@ class App(models.CraftBaseModel):
     daemon: Optional[Literal["simple", "forking", "oneshot", "notify", "dbus"]]
     after: UniqueStrList = cast(UniqueStrList, [])
     before: UniqueStrList = cast(UniqueStrList, [])
-    refresh_mode: Optional[Literal["endure", "restart"]]
+    refresh_mode: Optional[Literal["endure", "restart", "ignore-running"]]
```

Comment:
BTW at a later point we could turn many of these literals into enum values, which would have a convenient side-effect of having docstrings that can be included in auto-generated jsonschemas